src/renderers/shaders/ShaderLib.js

```diff
@@ -103,6 +103,36 @@ THREE.ShaderLib = {
 
 	},
 
+	'physical': {
+
+		uniforms: THREE.UniformsUtils.merge( [
```

Comment:
could potentially merge in the parameters from 'standard'?  Reduce deuplication?